doc/eclipse.html

```diff
@@ -50,13 +50,19 @@ <h2 align="center">TestNG Eclipse plug-in</h2>
 <br>
 &nbsp;&nbsp;<a class="summary" href="#eclipse-project-properties">9.2 - Project Properties</a>
 <br>
-&nbsp;&nbsp;<a class="summary" href="#testng-xml"></a>
+&nbsp;&nbsp;<a class="summary" href="#eclipse-m2e-integration">10 - M2E Integration</a>
 
 <!--
   Installation
 -->
 <h3><a name="eclipse-installation">1 - Installation</a></h3><p>
-Once you have <a href="http://testng.org/doc/download.html">installed the plug-in</a>, restart Eclipse and select the menu <tt>Window / Show View 
+Follow the instructions to <a href="http://testng.org/doc/download.html">install the plug-in</a> <br/>
+NOTE: since TestNG Eclipse Plugin 6.9.10, there is a new optinal plugin for M2E (Maven Eclipse Plugin) integration, it's recommended to install it if your Java project(s) are Maven managed.
```

Comment:
NOTE: since TestNG Eclipse Plugin 6.9.10, there is a new optional plug-in for M2E (Maven Eclipse Plugin) integration. It's recommended to install it if your Java project(s) are managed by Maven.